migration-engine/connectors/sql-migration-connector/src/flavour/mysql.rs

```diff
@@ -221,10 +227,24 @@ impl SqlFlavour for MysqlFlavour {
             return Err(SystemDatabase(db_name.to_owned()).into());
         }
 
-        let version = connection.version().await?;
+        let version = connection
+            .query_raw("SELECT @@version", &[])
+            .await?
+            .into_iter()
+            .next()
+            .and_then(|r| r.into_iter().next())
+            .and_then(|val| val.into_string());
+
+        let global_version = connection.version().await?;
```

Comment:
If we want to micro-optimize, we could even fetch the two version strings in one query :p

Reply:
I'd like to not change Quaint APIs for this now though.

Reply:
I meant by copying the query quaint does (probably `@@global.version`, so `SELECT @@version, @@global.version`)